src/output_settings.rs

```diff
@@ -54,6 +67,7 @@ pub struct OutputSettingsBuilder {
     scale: Option<u32>,
     pixel_spacing: Option<u32>,
     theme: BinaryColorTheme,
+    pixel_shape: Option<PixelShape>,
```

Comment:
There is no need to use an option for the pixel shape, because `Square` is the default:
```suggestion
    pixel_shape: PixelShape,
```